wagtail/users/forms.py

```diff
@@ -385,7 +385,12 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = UserProfile
-        fields = ("submitted_notifications", "approved_notifications", "rejected_notifications")
+        fields = ['submitted_notifications', 'approved_notifications', 'rejected_notifications']
```

Comment:
Why has this line changed from a tuple to a list?

Reply:
Because it should be a list!

Reply:
Haha, fair enough!